bin/roave-backward-compatibility-check.php

```diff
@@ -51,7 +52,7 @@
         throw new RuntimeException('Could not find Composer autoload.php');
     })();
 
-    $application = new Application();
+    $application = new Application('roave/backward-compatibility-check', Versions::getVersion('roave/backward-compatibility-check'));
```

Comment:
does this also work in a `Phar` environment?

Reply:
If the phar is built after `composer install` was performed, then yes. The `Versions` class will already have been generated at that point.

Reply:
I'd imagine it would be so :+1: